src/webextension/background.js

```diff
@@ -14,3 +14,92 @@ function openLockbox() {
 }
 
 browser.browserAction.onClicked.addListener(openLockbox);
+
+function makeEntrySummary(entry) {
+  return {site: entry.site, id: entry.id};
+}
+
+const fakeStore = {
```

Comment:
note the "real" DataStore is mostly async.  Not sure what kind of impact that has on the (not-quite-)reducer below ...